docs/preview/features/message-pumps/service-bus.md

```diff
@@ -302,6 +303,58 @@ string transactionId = correlationInfo.TransactionId;
 string operationId = correlationInfo.OperationId;
 ```
 
+## Automatic Azure Key Vault credentials rotation
+
+The additional library `Arcus.Messaging.Pumps.ServiceBus.KeyRotation` provides an extension on the message pump to restart the pump automatically when the credentials of the pump stored in Azure Key Vault are changed.
+This feature allows more reliable restarting instead of relying on authentication exceptions that may be throwed during the lifetime of the message pump.
+
+## How does this work?
+
+A background job will be run in the background which is in fact a message pump on a Service Bus Topic.
+The Azure Key Vault where the connection string for your target message pump is located should send `SecretNewVersionCreated` events to this Service Bus Topic.
```

Comment:
What about this?
```suggestion
A background job is polling for `SecretNewVersionCreated` events on an Azure Service Bus Topic for the secret that stores the connection string.
```

Reply:
The reason is that the rest of it is an implementation detail that people don't care about, imo